google-cloud-firestore/src/test/java/com/google/cloud/firestore/BulkWriterTest.java

```diff
@@ -675,7 +675,7 @@ public boolean onError(BulkWriterException error) {
   }
 
   @Test
-  public void retriesMaintainCorrectWriteResolutionOrdering() throws Exception {
+  public void retryResolvesBeforeFlush() throws Exception {
```

Comment:
I don't think I will be able to deflake the original test as the order of whether doc1, doc2, doc1 or doc1, doc1, doc2 is enqueued is no longer deterministic. I don't think that's a problem though, but it breaks this test.

Reply:
This behavior is still reasonable though, so it should be fine.